docs/packaging-an-application/config-screen.md

```diff
@@ -43,6 +43,18 @@ and `type` properties. Specific item types can including new types.
 
 ## Examples
 
+### `bool`
+The `bool` input type will accept a default value of "0" or "1"
```

Comment:
I think this should cover more '... input type should use a "0" or "1" to set the value'